### PR TITLE
Add Ansible

### PR DESCRIPTION
**Pull request by babbaranish**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

File: a/Ansible.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# This playbook prints a simple debug message
+- name: Echo 
+  hosts: 127.0.0.1
+  connection: local
+
+  tasks:
+  - name: Print Hello World!
```
Comment (Richienb) [inline a/Ansible.yml:8]:
```suggestion
  - name: Print Hello World
```

File: a/Ansible.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+# This playbook prints a simple debug message
+- name: Echo
```
Comment (Richienb) [inline a/Ansible.yml:3]: Inconsistently named with line 8, should it be called `Echo` or `Print`?

File: a/Ansible.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+# This playbook prints a simple debug message
```
Comment (Richienb) [inline a/Ansible.yml:2]: Unnecessary comment.

**Review by Richienb — CHANGES_REQUESTED** [review]

File: a/Ansible.yml (hunk 2 above)
Comment (babbaranish) [reply a/Ansible.yml]: changed it to print and removed comments too

**Review by Richienb — APPROVED** [review]

**Pull request merged**